generate_grpc.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+set -e
+
+# check and download dependency for gRPC code generate
+if [ ! -e ./grpc/proto/vendor/protobuf/src/google/protobuf ]; then
+    DIR="./grpc/proto/vendor/protobuf/src/google/protobuf"
+    rm -rf $DIR
+    mkdir -p $DIR
+    wget https://raw.githubusercontent.com/protocolbuffers/protobuf/v3.9.0/src/google/protobuf/empty.proto -P $DIR
```

Comment:
let's use `https://raw.githubusercontent.com/protocolbuffers/protobuf/v24.3/src/google/protobuf/empty.proto` 

all repos will be changed to use this newer version. There are two PRs for instance manager and engine going to merge soon.

Reply:
cc @FrankYang0529 is working on it.